imagepaste/clipboard/darwin/darwin.py

```diff
@@ -65,7 +66,8 @@ def push(cls, save_directory: str) -> DarwinClipboard:
                 "close access pastedImage",
             ]
             process = Process.execute(cls.get_osascript_args(commands))
-            if process.stderr:
+
+            if not isfile(filepath):
                 return cls(Report(3, f"Cannot save image: {image} ({process.stderr})"))
             return cls(Report(6, f"Saved and pasted 1 image: {image}"), [image])
         return cls(Report(3))
```

Comment:
In the old code this line is `return 1` equivalent to `Unable to save image`, can you explain this?